api/twilioapi/twilioapi.py

```diff
@@ -40,49 +42,48 @@ async def voice(
         CallSid: str = Form()
 ):
     response = VoiceResponse()
+    global conv_engine
     if From in data["trusted_group"]:
-        response.say("Redirecting to recipient. Please wait a moment")
+        response.say("Redirecting to recipient. Please wait a moment", voice=voice_type)
         response.dial(data["recipient"])
         return Response(content=str(response), headers={"Content-Type": "text/xml"})
-    if From == "anonymous":
-        pass
-        # conv_engine = ConversationFactory.hidden(CallSid)
-        # response.say(conv_engine.start_message)
-        # response.gather(input='speech', action=f'{config.PUBLIC_URL}/api/twilio/gather_hidden', timeout=2)
-        # return Response(content=str(response), headers={"Content-Type": "text/xml"})
+    if From == "anonymous" or From == "Anonymous":
```

Comment:
I see `from` is a reserved keyword.
I was a bit confused reading

`From: str = From(),`

Architecture-wise, the code in this function is like a controller.